examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -406,4 +409,16 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback {
         return PreferenceManager.getDefaultSharedPreferences(this.applicationContext)
                 .getBoolean(this.getString(R.string.simulate_route_key), false)
     }
+
+    private fun updatePuck(routeProgress: RouteProgress) {
```

Comment:
Shouldn't this logic moved into the SDK side? I believe that should be handled internally and allow developers to customize colors (👀 puck customization options currently exposed from the Navigation SDK).

Reply:
I agree this logic should be handled by the UI SDK, not the customer.